registration/include/pcl/registration/registration.h

```diff
@@ -609,6 +609,12 @@ namespace pcl
     private:
       /** \brief The point representation used (internal). */
       PointRepresentationConstPtr point_representation_;
+
+      /**
+       * \brief Remove from public API in favor of \ref setInputSource
+       */
+      using PCLBase<PointSource>::setInputCloud;
```

Comment:
Semantically, [deleting the method](https://en.cppreference.com/w/cpp/language/function#Deleted_functions) would better express the right intent here. Move it to the original scope and delete it.

Reply:
https://godbolt.org/z/qYpD-u

Can't delete a virtual function, apparently. Any other syntax that can help achieve this?